test/ring/middleware/oauth2_test.clj

```diff
@@ -229,3 +229,17 @@
         (is (= "/" (get-in response [:headers "Location"])))
         (is (approx-eq (-> 3600 time/seconds time/from-now)
                        (-> response :session ::oauth2/access-tokens :test :expires)))))))
+
+(defn redirect-handler [{:keys [oauth2/access-tokens]}]
+  {:status 200, :headers {}, :body "redirect-handler-response-body"})
+
+(deftest test-redirect-handler
+  (let [profile  (assoc test-profile
```

Comment:
Can you align variables and values in this `let` form? The spacing you've added seems random.

Reply:
Fixed format.